README.md

```diff
@@ -1,3 +1,24 @@
-# akka-cluster-management
+# Akka Cluster Management
 
-This repository contains interfaces to interact with an Akka Cluster.
\ No newline at end of file
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/akka/akka?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+This repository contains interfaces to interact with an Akka Cluster.
+
+Documentation
+-------------
+
+See [reference](http://developer.lightbend.com/docs/akka-cluster-management/0.1/) and [API](http://developer.lightbend.com/docs/api/akka-cluster-management/0.3/akka/cluster/http/management/index.html) documentation pages.
```

Comment:
The sentence is "reference and documentation pages ... ". This is copied from Alpakka...